awx/api/serializers.py

```diff
@@ -141,7 +141,7 @@
     'target_credential': DEFAULT_SUMMARY_FIELDS + ('kind', 'cloud', 'credential_type_id'),
     'webhook_credential': DEFAULT_SUMMARY_FIELDS,
     'approved_or_denied_by': ('id', 'username', 'first_name', 'last_name'),
-    'credential_type': DEFAULT_SUMMARY_FIELDS,
+    'credential_type': DEFAULT_SUMMARY_FIELDS + ('managed_by_tower',),
```

Comment:
I think this is a good idea because the UI needs to behave differently for different credentials based on whether or not they're `managed_by_tower`. We recently added `credential_type` as a `SUMMARIZABLE_FK_FIELD` and we probably should've included this field then.

cc @ryanpetrello @matburt